### PR TITLE
Fixed Issue: #2822 404 page tells users to contact webmaster@mozilla.com

### PR DESCRIPTION
**Pull request by lavish205**
#2822

### CODE REVIEW

**Comment by eviljeff** [comment]
I'd have `please <a href="https://github.com/mozilla/addons-server/issues/new/">file an issue</a>`.

**Comment by muffinresearch** [comment]
@lavish205 as @eviljeff has pointed out it will be good to have more context to the link rather than linking a word like "here". More context helps with accessibility.

This whole file is translated so we'll need to tip off the l10ners when this lands so that we can expedite the translations being updated.

**Comment by muffinresearch** [comment]
We also need to enable fuzzy matching on the merging post-extraction. I'll file a separate issue for that.

**Comment by lavish205** [comment]
Updated

File: src/olympia/amo/templates/amo/404.html [hunk 1]
```diff
@@ -14,8 +14,8 @@
 <ul>
   <li>If you typed in the address, please double check the spelling.</li>
   <li>
-    If you followed a link from somewhere, please let us know at
-    <a href="mailto:webmaster@mozilla.com">webmaster@mozilla.com</a>.
+    If you followed a link from somewhere, please
```
Comment (wagnerand) [inline src/olympia/amo/templates/amo/404.html]: Nit: Can you please remove the second space?

**Comment by eviljeff** [comment]
r+

**Comment by lavish205** [comment]
Removed extra space. please review.

**Comment by wagnerand** [comment]
r+ , thanks!

**Pull request merged**